### ISSUE TITLE
Pre-work Checklist: Developer: Vanessa Vun

### ISSUE THREAD

**Issue by vanessavun** [hackforla/website#3914]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by vanessavun** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings? ~6.5-7.5 hours.

Pre-work up such as installing Docker took about 3-4 active hours because I had to check for WSL2, install WSL2, enable virtualization in my desktop CPU (Windows 11), and reinstall Docker a few times. Cloning the Github repo and editing the website files did not take too long.

My onboarding took 3.5 hours. I did not attend the first meetings but I will attend next week.

**Comment by vanessavun** [comment]
Provide Update
1. Progress: finished my 1st issue and now waiting for pull request review: https://github.com/hackforla/website/pull/3932
2. Blockers: no blockers as I have already received help through Slack.
3. Availability: upcoming week, all of Friday
4. ETA: done.

**Comment by HackforLABot** [comment]
## Activity Log: vanessavun
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- vanessavun opened: https://github.com/hackforla/website/issues/3914 at 01/31/2023, 08:58 PM PST
- vanessavun assigned: https://github.com/hackforla/website/issues/3914 at 01/31/2023, 08:58 PM PST
- vanessavun assigned: https://github.com/hackforla/website/issues/3837#issuecomment-1409525426 at 02/02/2023, 11:54 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3837#issuecomment-1415268941 at 02/02/2023, 11:54 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3837#issuecomment-1416369681 at 02/03/2023, 12:30 PM PST
- vanessavun opened: https://github.com/hackforla/website/pull/3932 at 02/03/2023, 04:39 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3914#issuecomment-1416569976 at 02/03/2023, 05:00 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3914#issuecomment-1416572750 at 02/03/2023, 05:06 PM PST
- vanessavun closed by PR 3940: https://github.com/hackforla/website/issues/3854#event-8444409867 at 02/05/2023, 11:33 PM PST
- vanessavun reopened: https://github.com/hackforla/website/issues/3854#event-8444409867 at 02/05/2023, 11:34 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3854#issuecomment-1418637447 at 02/05/2023, 11:36 PM PST
- vanessavun PR was merged: https://github.com/hackforla/website/pull/3932#event-8453046992 at 02/06/2023, 04:37 PM PST
- vanessavun issue closed by PR 3932: https://github.com/hackforla/website/issues/3837#event-8453047120 at 02/06/2023, 04:37 PM PST
- vanessavun assigned: https://github.com/hackforla/website/issues/3540#issuecomment-1397449627 at 02/08/2023, 04:45 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3540#issuecomment-1423445651 at 02/08/2023, 04:46 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3540#issuecomment-1427127754 at 02/12/2023, 12:41 PM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/3964#issuecomment-1427128455 at 02/12/2023, 12:45 PM PST
- vanessavun submitted review: https://github.com/hackforla/website/pull/3964#pullrequestreview-1296269794 at 02/13/2023, 11:01 AM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/3979#issuecomment-1431984671 at 02/15/2023, 12:25 PM PST
- vanessavun submitted review: https://github.com/hackforla/website/pull/3979#pullrequestreview-1300304658 at 02/15/2023, 12:30 PM PST
- vanessavun closed as completed: https://github.com/hackforla/website/issues/3914#event-8530232101 at 02/15/2023, 12:41 PM PST
- vanessavun assigned: https://github.com/hackforla/website/issues/3822 at 02/16/2023, 08:20 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3822#issuecomment-1434084919 at 02/16/2023, 08:21 PM PST
- vanessavun opened: https://github.com/hackforla/website/pull/3995 at 02/16/2023, 08:42 PM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/3989#issuecomment-1434099151 at 02/16/2023, 08:52 PM PST
- vanessavun submitted review: https://github.com/hackforla/website/pull/3989#pullrequestreview-1302692597 at 02/16/2023, 08:56 PM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/3993#issuecomment-1434103312 at 02/16/2023, 08:59 PM PST
- vanessavun submitted review: https://github.com/hackforla/website/pull/3993#pullrequestreview-1302700704 at 02/16/2023, 09:10 PM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/3994#issuecomment-1434122837 at 02/16/2023, 09:29 PM PST
- vanessavun submitted review: https://github.com/hackforla/website/pull/3994#pullrequestreview-1302723724 at 02/16/2023, 09:39 PM PST
- vanessavun PR was merged: https://github.com/hackforla/website/pull/3995#event-8552395381 at 02/17/2023, 09:24 PM PST
- vanessavun issue closed by PR 3995: https://github.com/hackforla/website/issues/3822#event-8552395409 at 02/17/2023, 09:24 PM PST
- vanessavun assigned: https://github.com/hackforla/website/issues/3879 at 02/19/2023, 02:04 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3879#issuecomment-1436109344 at 02/19/2023, 02:05 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3540#issuecomment-1436111601 at 02/19/2023, 02:14 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3879#issuecomment-1439448745 at 02/21/2023, 09:10 PM PST
- vanessavun submitted review: https://github.com/hackforla/website/pull/3979#pullrequestreview-1312409083 at 02/23/2023, 06:21 PM PST
- vanessavun issue closed as completed: https://github.com/hackforla/website/issues/3540#event-8597998388 at 02/23/2023, 08:17 PM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/4032#issuecomment-1442825976 at 02/23/2023, 09:21 PM PST
- vanessavun submitted review: https://github.com/hackforla/website/pull/4032#pullrequestreview-1312531764 at 02/23/2023, 09:25 PM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/3964#issuecomment-1442830181 at 02/23/2023, 09:28 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3879#issuecomment-1442853097 at 02/23/2023, 10:05 PM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/3979#issuecomment-1444244237 at 02/24/2023, 10:45 AM PST
- vanessavun submitted review: https://github.com/hackforla/website/pull/3979#pullrequestreview-1314463463 at 02/24/2023, 11:08 PM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/3979#issuecomment-1445017144 at 02/24/2023, 11:09 PM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/4038#issuecomment-1445018815 at 02/24/2023, 11:16 PM PST
- vanessavun submitted review: https://github.com/hackforla/website/pull/4038#pullrequestreview-1314464472 at 02/24/2023, 11:19 PM PST
- vanessavun unassigned: https://github.com/hackforla/website/issues/3879#issuecomment-1442853097 at 02/25/2023, 09:04 AM PST
- vanessavun assigned: https://github.com/hackforla/website/issues/3879#issuecomment-1442853097 at 02/25/2023, 09:04 AM PST
- vanessavun issue closed as completed: https://github.com/hackforla/website/issues/3879#event-8608196447 at 02/25/2023, 09:08 AM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3879#issuecomment-1447710653 at 02/27/2023, 11:35 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3837#issuecomment-1448446187 at 02/28/2023, 08:06 AM PST
- vanessavun assigned: https://github.com/hackforla/website/issues/3959#issuecomment-1445715829 at 02/28/2023, 11:26 AM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3959#issuecomment-1448738828 at 02/28/2023, 11:26 AM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/4053#issuecomment-1448740978 at 02/28/2023, 11:28 AM PST
- vanessavun submitted review: https://github.com/hackforla/website/pull/4053#pullrequestreview-1319068431 at 02/28/2023, 09:34 PM PST
- vanessavun commented on: https://github.com/hackforla/website/pull/4053#issuecomment-1449380031 at 02/28/2023, 09:37 PM PST
- vanessavun unassigned: https://github.com/hackforla/website/issues/3959#issuecomment-1450872479 at 03/03/2023, 03:17 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3959#issuecomment-1454253251 at 03/03/2023, 03:18 PM PST
- vanessavun assigned: https://github.com/hackforla/website/issues/3959#issuecomment-1454253251 at 03/09/2023, 01:08 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3959#issuecomment-1462831584 at 03/09/2023, 01:12 PM PST
- vanessavun commented on: https://github.com/hackforla/website/issues/3959#issuecomment-1473255525 at 03/16/2023, 11:44 PM PDT
- vanessavun assigned: https://github.com/hackforla/website/issues/3636#issuecomment-1469030676 at 03/19/2023, 11:58 AM PDT
- vanessavun unassigned: https://github.com/hackforla/website/issues/3636#issuecomment-1475362180 at 03/19/2023, 12:02 PM PDT
- vanessavun unassigned: https://github.com/hackforla/website/issues/3959#issuecomment-1473255525 at 03/20/2023, 03:59 PM PDT
- vanessavun commented on: https://github.com/hackforla/website/issues/3959#issuecomment-1477059261 at 03/20/2023, 04:00 PM PDT
- vanessavun assigned: https://github.com/hackforla/website/issues/3636#issuecomment-1484153512 at 03/26/2023, 10:27 AM PDT
- vanessavun unassigned: https://github.com/hackforla/website/issues/3636#issuecomment-1484153512 at 03/26/2023, 01:34 PM PDT
- vanessavun commented on: https://github.com/hackforla/website/pull/4517#issuecomment-1509558406 at 04/14/2023, 11:06 PM PDT
- vanessavun submitted review: https://github.com/hackforla/website/pull/4517#pullrequestreview-1386324524 at 04/14/2023, 11:17 PM PDT
- vanessavun submitted review: https://github.com/hackforla/website/pull/4517#pullrequestreview-1389189296 at 04/17/2023, 07:49 PM PDT
- vanessavun commented on: https://github.com/hackforla/website/pull/4539#issuecomment-1512362538 at 04/17/2023, 07:56 PM PDT
- vanessavun submitted review: https://github.com/hackforla/website/pull/4539#pullrequestreview-1389196885 at 04/17/2023, 08:02 PM PDT
- vanessavun commented on: https://github.com/hackforla/website/pull/4561#issuecomment-1521015103 at 04/24/2023, 05:59 PM PDT
- vanessavun submitted review: https://github.com/hackforla/website/pull/4561#pullrequestreview-1398988960 at 04/24/2023, 06:04 PM PDT
